doc/index.rst

```diff
@@ -57,16 +57,20 @@ Documentation
 See also
 --------
 
+- Stephan Hoyer and Joe Hamman's `Journal of Open Research Software paper`_ describing the xarray project.
 - Stephan Hoyer's `SciPy2015 talk`_ introducing xarray to a general audience.
 - Stephan Hoyer's `2015 Unidata Users Workshop talk`_ and `tutorial`_ (`with answers`_) introducing
   xarray to users familiar with netCDF.
 - `Nicolas Fauchereau's tutorial`_ on xarray for netCDF users.
+- The `UW eScience Institute's Geohackweek`_ tutorial on xarray for geospatial data scientists.
```

Comment:
Maybe give this tutorial prominence over the others? 

Some of the other links might need some cleaning and/or updating, but this could happen in a different PR